homeassistant/components/synology_dsm/strings.json

```diff
@@ -1,18 +1,30 @@
 {
   "config": {
     "title": "Synology DSM",
+    "flow_title": "Synology DSM {name} ({host})",
     "step": {
       "user": {
         "title": "Synology DSM",
         "data": {
           "name": "Name",
```

Comment:
Did we drop name in the user form?

Reply:
Yes, but its still displayed in the title so its easy to figure out which one you are setting up

Reply:
Ok. But we should remove this string, right?

Reply:
If there are `placeholders` and no `follow_title` it will show an empty string in discovery.  I found this out the hard way in #33557

Reply:
I'm referring to line 9.

Reply:
> I'm referring to line 9.

Well yeah, that needs to come out :)